yokadi/tests/ydateutilstestcase.py

```diff
@@ -123,4 +123,14 @@ def testParseDateLimit(self):
             self.assertEqual(expectedOp, output[0])
             self.assertEqual(expectedDate, output[1])
 
+    def testGuessTime(self):
+        for invalidTime in ("+5M", "+1m", "+2H", "+3h", "+9D", "+14d", "+432W", "+0w",
+                     "01/01/2009", "10/10/2008 12", "7/7/2007 10:15", "1/2/2003 1:2:3"):
+            self.assertIsNone(ydateutils.guessTime(invalidTime))
+
+
+        for text, expected_result in (('12:05:20', time(hour=12, minute=5, second=20)), ('10:00am', time(hour=10)), ('7:30pm', time(hour=19, minute=30))):
```

Comment:
Nitpick: Yokadi uses camelCase for variables, so "expectedResult" (one day, I will PEP8-ify it :))